### PR TITLE
RELENG-2986 applying a 0 burst policy

### PR DESCRIPTION
**Pull request by tcarmet**
TL;DR bumping cpu and memory resources on the CI

On CI workloads, you cannot guess the state of the node you're going
to schedule in, therefore, for functional tests it is advised to not do
any burst on resources.

Here you'll see the requested VS what is actually beeing used for CPU resources on one build
image
note that we use a lot more than we ask for, which is the info that the kubernetes node scheduler use to schedule the pods, and what I'm afraid is that we might schedule in some builds our workload on a already stressed node.

Therefore the 0 burst policy, we request what we need, nothing more, nothing less.

This is what the same chart looks like with the builds from now on.
image

### CODE REVIEW

**Comment by tcarmet** [comment]
@bert-e wait

**Review by jonathan-gramain — APPROVED** [review]

**Review by JianqinWang — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

**Review by ssalaues — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Comment by tcarmet** [comment]
@bert-e approve

**Review by dora-korpar — APPROVED** [review]
